src/taco-dependency-installer/installers/androidPackagesInstaller.ts

```diff
@@ -77,26 +82,25 @@ class AndroidPackagesInstaller extends InstallerBase {
         }
     }
 
-    private getAvailableAndroidPackages(): Q.Promise<any> {
-        var deferred: Q.Deferred<any> = Q.defer();
+    private executeAndroidCommand(name: string, args: string[]): IExecuteCommandResult {
```

Comment:
Regarding the second comment, what do you think of this signature?:
executeAndroidCommand(args: string[], nameForTelemetry: string);

telemetryStep has the connotation of telemetry.step(); and we are not doing that, so I'd like to use a different concept.
Regarding the optional value, the telemetry name is quite important for us to use telemetry effectively, and I think that whoever calls this should spend some time thinking of a good name for this, so I want it to be mandatory.